roles/zuul/files/jobs/lore.yaml

```diff
@@ -8,7 +8,8 @@
           #!/bin/bash -ex
           sudo apt-get install -yqq curl
           curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash
-          source ~/.bashrc
+          export NVM_DIR="/home/bonnyci/.nvm"
+          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
```

Comment:
Are we sure we want to do this instead of just installing node via APT (BonnyCI/hoist#94)?